tools/buildsteps/win32/bootstrap-addons.bat

```diff
@@ -32,6 +35,7 @@ IF "%WORKDIR%" == "" (
 )
 
 rem setup some paths that we need later
+SET BUILD_ON_CORES=8
```

Comment:
Ah didn't know that. You mean %NUMBER_OF_PROCESSORS%?
Which is discussed [here](http://stackoverflow.com/questions/22919076/find-number-of-cpus-and-cores-per-cpu-using-command-prompt).